std_msgs/Header header
string robot_name
geometry_msgs/Transform world_to_base_transform
string[] joint_name
float64[] joint_position
float64[] joint_velocity
float64[] joint_external_torque
float64[] joint_measured_torque
